# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/html/HTMLTableAccessible.h
# Commit: 6d0f4821ddcf
# Full Hash: 6d0f4821ddcfca1b59e4db3befc3ec9c12335891
# Author: James Teh <jteh@mozilla.com>
# Date: 2022-03-23 09:49:32
# Regressor Bug: 1735970
# File Overlap Count: 1
# Description:
#   Bug 1735970 part 3: Add TableAccessibleBase and TableCellAccessibleBase. r=morgan
#   
#   These are needed to support RemoteAccessible tables.
#   Stuff specific to LocalAccessible is still in TableAccessible and TableCellAccessible, which now inherit from the new Base classes.
#   
# ==============================================================================

diff -r d82660fb5408 -r 6d0f4821ddcf accessible/html/HTMLTableAccessible.h
--- a/accessible/html/HTMLTableAccessible.h	Wed Mar 23 04:00:25 2022 +0000
+++ b/accessible/html/HTMLTableAccessible.h	Wed Mar 23 04:00:26 2022 +0000
@@ -52,8 +52,8 @@
   virtual uint32_t RowIdx() const override;
   virtual uint32_t ColExtent() const override;
   virtual uint32_t RowExtent() const override;
-  virtual void ColHeaderCells(nsTArray<LocalAccessible*>* aCells) override;
-  virtual void RowHeaderCells(nsTArray<LocalAccessible*>* aCells) override;
+  virtual void ColHeaderCells(nsTArray<Accessible*>* aCells) override;
+  virtual void RowHeaderCells(nsTArray<Accessible*>* aCells) override;
   virtual bool Selected() override;
 
  protected:
@@ -152,7 +152,7 @@
   virtual uint32_t SelectedCellCount() override;
   virtual uint32_t SelectedColCount() override;
   virtual uint32_t SelectedRowCount() override;
-  virtual void SelectedCells(nsTArray<LocalAccessible*>* aCells) override;
+  virtual void SelectedCells(nsTArray<Accessible*>* aCells) override;
   virtual void SelectedCellIndices(nsTArray<uint32_t>* aCells) override;
   virtual void SelectedColIndices(nsTArray<uint32_t>* aCols) override;
   virtual void SelectedRowIndices(nsTArray<uint32_t>* aRows) override;